services/src/main/java/org/keycloak/exportimport/singlefile/SingleFileExportProvider.java

```diff
@@ -96,6 +96,13 @@ private ObjectMapper getObjectMapper() {
     }
 
     private void writeToFile(Object reps) throws IOException {
+
+        if (!file.getParentFile().exists()) {
```

Comment:
Needed this because for some reason the parent folder for:
`/home/tom/dev/repos/gh/thomasdarimont/keycloak-dev/keycloak/testsuite/integration-arquillian/tests/base/target/export-import/singleFile-realm.json`
were not created during tests...